common/include/pcl/point_cloud.h

```diff
@@ -648,6 +812,19 @@ namespace pcl
         return (it);
       }
 
+      /** \brief Erase a set of points given by a (first, last) iterator pair
+        * \note This assumes the user would correct the details later on!
+        * \param[in] first where to start erasing points from
+        * \param[in] last where to stop erasing points from
+        * \return returns the new position iterator
+        */
+      inline iterator
+      transient_erase (iterator first, iterator last)
+      {
+        iterator it = points.erase (first, last);
```

Comment:
should/could std::move be used here on the iterators, like with the assign method?

Reply:
Updated